app/javascript/src/components/editor/TranslationKeys/TranslationKeysEditStrings.svelte

```diff
@@ -4,20 +4,32 @@
   import { submittable } from "@components/actions/submittable"
   import { createEventDispatcher } from "svelte"
   import debounce from "@src/debounce"
+  import { createNewTranslationKey } from "@src/lib/translations"
 
   export let selectedKey
 
   const dispatch = createEventDispatcher()
 
-  let renameInput
+  let createNewKeyMode
+  let createOrRenameInput
   let error = ""
+  let selectedKeyNameInExamples
 
-  const renameKey = debounce(() => {
+  $: {
+    createNewKeyMode = selectedKey === createNewTranslationKey
+
+    selectedKeyNameInExamples = createNewKeyMode ? "…" : selectedKey
+
+    if (createNewKeyMode && createOrRenameInput)
+      createOrRenameInput.focus()
+  }
```

Comment:
I'm personally not the biggest fan of big reactive statements like this. I prefer to keep it to one line if possible and extract the rest to a function, that way you have better control over what is and isn't reactive

Reply:
How about this? 4bf02b78c4011e1d5f3bc7c5d304ff3ba05df249

Reply:
Nice, that's much more readable